PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,9 @@
 <!--
 Thanks for contributing a pull request! Please ensure you have taken a look at
-the contribution guidelines: https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#Contributing-Pull-Requests
+the contribution guidelines: http://scikit-learn.org/dev/developers/contributing.html#contributing-pull-requests
 -->
-#### Reference Issue
-<!-- Example: Fixes #1234 -->
+#### Reference Issues/PRs
+<!-- Example: Fixes #1234 / Closes #2345 / See also #3456 -->
```

Comment:
I was quite intentional in using . not / in my comment, to suggest that these can be used together. Is there a way that this can be clearer?

Reply:
@jnothman Sorry for not reading carefully for your comment.
From my perspective, "Example: Closes(Fixes?) #1234. Closes #2345. See also #3456." make me feel a bit confused. For more complex situation, seems that it might be better to mention in the contribution guide.
My original intention here is to add Closes here, since sometimes contributors may not feel that they are fixing something (e.g., continue a PR) and they can use Closes. So, I might use "Example: Fixes #1234 / Closes #2345" here.
Maybe we can add something like "If your PR is simply related to some other issues/PRs, create a link to them without using the keywords (e.g., See also #1234)" in contribution guide, but I'm not sure how useful it will be.
WDYT? I'll follow your final decision. Thanks :)